src/google-classroom-extractor/edfi_google_classroom_extractor/api/api_caller.py

```diff
@@ -23,7 +24,7 @@
 
 
 @retry(
-    retry_on_exceptions=(IOError, RequestException, GoogleApiError),
+    retry_on_exceptions=(IOError, RequestException, GoogleApiError, socket.timeout, socket.error),
```

Comment:
Should these be added to Canvas and Schoology as well? If so, can you create a ticket for this modification?

Reply:
Will do